src/csvcubed/models/rdf/__init__.py

```diff
@@ -1,3 +0,0 @@
-"""
-Contains Models for mapping data to RDF which are unique to the `csvcubed` package.
```

Comment:
Let's keep any comments we have. It's only the imports that we're planning to get rid of.

Reply:
Yeah My bad I took it literally